modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java

```diff
@@ -268,9 +313,21 @@ private DefaultCouchbaseEnvironment createCouchbaseEnvironment() {
         initCluster();
         return DefaultCouchbaseEnvironment.builder()
             .bootstrapCarrierDirectPort(getMappedPort(11210))
```

Comment:
I think we also need to have the ssl ports mapped (carrier and http)

Reply:
we thought about it, but do users really run tests again SSL-protected DBs?